cfme/tests/cloud/test_keypairs.py

```diff
@@ -38,8 +39,7 @@ def test_keypair_crud(openstack_provider):
             pytest.skip('Timed out creating keypair, BZ1444520')
         else:
             pytest.fail('Timed out creating keypair')
-    view = navigate_to(keypair, 'Details')
-    assert view.is_displayed
+    keypair.exists()
```

Comment:
nice! Good enhancement to the class. I'd still assert keypair.exists() here though, since if it returns false the test won't record it.